operations/utility_storage/Remove_Ceph_Node.md

```diff
@@ -94,69 +92,68 @@ This procedure describes how to remove a Ceph node from the Ceph cluster. Once t
+        ncn-s001# pdcp -w ncn-s00[2-(end node number)] /etc/haproxy/haproxy.cfg /etc/haproxy/haproxy.cfg
         ```
 
-   3. Restart the services on all the storage nodes and stop on the node that is being removed.
+   1. Restart the services on all the storage nodes, and stop it on the node that is being removed.
 
         ```bash
-        pdsh -w ncn-s00[1-(end node number)] -f 2 'systemctl restart haproxy.service'
-        pdsh -w $NODE 'systemctl stop haproxy.service'
+        ncn# pdsh -w ncn-s00[1-(end node number)] -f 2 'systemctl restart haproxy.service'
+        ncn# pdsh -w $NODE 'systemctl stop haproxy.service'
         ```
 
-   
-    4. Redeploy the Rados Gateway containers to adjust the placement group.
+    1. Redeploy the Rados Gateway containers to adjust the placement group.
 
         ```bash
-        ceph orch apply rgw site1 zone1 --placement="<num-daemons> <node1 node2 node3 node4 ... >" --port=8080
+        ncn-ms# ceph orch apply rgw site1 zone1 --placement="<num-daemons> <node1 node2 node3 node4 ... >" --port=8080
         ```
 
         For example:
 
         ```bash
-        ceph orch apply rgw site1 zone1 --placement="3 ncn-s001 ncn-s002 ncn-s003" --port=8080
+        ncn-ms# ceph orch apply rgw site1 zone1 --placement="3 ncn-s001 ncn-s002 ncn-s003" --port=8080
         ```
 
-    5. Verify Rados Gateway is running on the desired nodes.
+    1. Verify the Rados Gateway is running on the desired nodes.
 
         ```bash
-        ceph orch ps --daemon_type rgw
+        ncn-ms# ceph orch ps --daemon_type rgw
         ```
 
         Example output:
 
-        ```bash
+        ```
         NAME                             HOST      STATUS         REFRESHED  AGE  VERSION  IMAGE NAME                         IMAGE ID      CONTAINER ID
         rgw.site1.zone1.ncn-s001.kvskqt  ncn-s001  running (41m)  6m ago     41m  15.2.8   registry.local/ceph/ceph:v15.2.8   553b0cb212c   6e323878db46
         rgw.site1.zone1.ncn-s002.tisuez  ncn-s002  running (41m)  6m ago     41m  15.2.8   registry.local/ceph/ceph:v15.2.8   553b0cb212c   278830a273d3
         rgw.site1.zone1.ncn-s003.nnwuqy  ncn-s003  running (41m)  6m ago     41m  15.2.8   registry.local/ceph/ceph:v15.2.8   553b0cb212c   a9706e6d7a69
         ```
   
-8. Remove the node from the cluster.
+1. Remove the node from the cluster.
 
-    Run from ncn-s001/2/3:
+    Run from `ncn-s001`, `ncn-s002`, or `ncn-s003`:
 
     ```bash
-    ceph orch host rm $NODE
+    ncn-s# ceph orch host rm $NODE
     ```
 
-9. Remove the Ceph configuration from the node.
+1. Remove the Ceph configuration from the node.
 
    On the ***node being removed***
 
     ```bash
-    cephadm rm-cluster --fsid $(cephadm ls|jq -r .[1].fsid) --force
+    ncn-s# cephadm rm-cluster --fsid $(cephadm ls|jq -r .[1].fsid) --force
     ```
 
-10. Remove the node from the crush map
+1. Remove the node from the crush map
```

Comment:
- This line is missing a period
- Change "crush" to "CRUSH"

Reply:
Both fixed